### PR TITLE
Send email sign-in link

### PR DESCRIPTION
**Pull request by jordynchuhaloff**
If you call main like this:
```js
main({
  email: "jordyn@dispostable.com",
  dynamicLinkSettings: {
    referrerId: "io.sunship.app",
    domain: "pasapesos.page.link",
    path: "auth-email",
  },
});
```
we will send an email with a link to sign in. Clicking the link will open Vibrant.

I will be updating Vibrant to intercept the link and pass it back into the recovery server screen. The recovery server will use it to finish the sign in process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,20 @@ dev tools:
 main({ phoneNumber: "+15551112222" });
 ```
 
+or
+
+```js
+main({
+  email: "jordyn@dispostable.com",
+  // Vibrant needs to pass in dynamicLinkSettings, they change depending on the env
```
Comment (fnando) [inline README.md:36]:
```suggestion
  // Client needs to pass in dynamicLinkSettings, they change depending on the env
```

**Review by fnando — APPROVED** [review]
Looking good! 👍

File: README.md [hunk 2]
```diff
@@ -28,6 +28,20 @@ dev tools:
 main({ phoneNumber: "+15551112222" });
 ```
 
+or
+
+```js
+main({
+  email: "jordyn@dispostable.com",
```
Comment (fnando) [inline README.md:35]:
```suggestion
  email: "jordyn@example.com",
```

**Comment by jordynchuhaloff** [comment]
thanks!

**Pull request merged**